apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js

```diff
@@ -180,14 +179,14 @@ export class NotifySubscribers {
 			const reference = [caseReference, update.id, subscription.id].join('-');
 			const projectLink = this.generateProjectLink(caseReference);
 			const unsubscribeUrl = this.generateUnsubscribeLink(caseReference, subscription.emailAddress);
-			const projectName = update.projectName || caseReference;
-
 			await this.notifyClient.sendEmail(this.templateId, subscription.emailAddress, {
 				personalisation: {
-					projectName,
+					// todo: project name & link ?
+					projectName: caseReference,
 					projectLink,
-					title: `${projectName} - Project Update`,
-					subject: `${projectName} - Project Update Notification`,
+					// todo: what should these be?
+					title: `${caseReference} - Project Update`,
+					subject: `${caseReference} - Project Update Notification`,
```

Comment:
The email needs to have the project name displayed as the title and subject, not the case reference, unless that has been reverted?

Reply:
As above, it shouldn't have been reverted.  Now restored.